catalog/security/catalog-security-ingestplugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,28 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0" xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+
+    <bean id="ingestPlugin" class="ddf.catalog.security.ingest.IngestPlugin">
+        <cm:managed-properties persistent-id="ddf.catalog.security.ingest.IngestPlugin"
+                               update-strategy="container-managed" />
+        <property name="permissionStrings">
+            <array>
+                <value>http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=anonymous</value>
+            </array>
+        </property>
+    </bean>
+
+    <service ref="ingestPlugin" auto-export="interfaces" ranking="1" />
```

Comment:
I was confusing this with filter ranking. I will change it to 0 like the others.